docs/instrumenting-rails.md

```diff
@@ -1,6 +1,45 @@
 # Instrumenting Rails
 
-TODO
+Install the instrumentation library by adding it to your project's `Gemfile`:
+
+``` ruby
+gem "opentelemetry-instrumentation-rails", "~> 0.24"
+```
+
+or install the gem using `bundle`:
+
+```shell
+bundle add opentelemetry-instrumentation-rails --version "~> 0.24"
+```
+
+Configure OpenTelemetry to use all available instrumentation libraries by adding the
+`Splunk::Otel` module from `splunk/otel` and `use_all()` method to a Rails initializer:
+
+``` ruby
+# config/initializers/opentelemetry.rb
```

Comment:
I think it's safer to go with `config/environment.rb` above `Rails.application.initialize!`.

Reply:
Why's that? I think we should have a really good reason to differ from the upstream docs on something like this. Happy to change it, just want to make sure.

Reply:
@jtmal-signalfx ping

Reply:
oops! somehow I missed the notification. Alright, we can follow upstream.